documentation/specs/src/interoperability/ethereum-bridge.md

```diff
@@ -30,363 +36,8 @@ Ethereum side.
  - [Proofs and validator set updates](./ethereum-bridge/proofs.md)
  - [Smart Contracts](./ethereum-bridge/ethereum_smart_contracts.md)
 
```

Comment:
I like this simplified introduction, but with this we should briefly mention what the bridge allows for (transfer of fungible & non-fungible tokens).